### PR TITLE
update Aaron's keys

### PR DESCRIPTION
**Pull request by aaronhaslett**

### CODE REVIEW

**Comment by aaronhaslett** [comment]
@stuartalexwhitehead your key breaking CI - not sure why

**Comment by ekini** [comment]
Most likely because it has expired.

**Comment by gdelavil** [comment]
Note that you an still merge once you have approvals

**Review by JohannDickson — APPROVED** [review]

**Pull request merged**
